extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -155,6 +163,13 @@ export class AmpImageViewer extends AMP.BaseElement {
     if (this.loadPromise_) {
       return this.loadPromise_;
     }
+    // TODO(sparhami) #19259 Tracks a more generic way to do this. Remove once
+    // we have something better.
+    const isScaled = closestAncestorElementBySelector(
+        this.element, '[i-amp-scale-animation]');
```

Comment:
Nit: the build checks don't like `i-amp-`, should be `i-amphtml-`.

Reply:
Done.